dvc/commands/get.py

```diff
@@ -14,12 +14,8 @@ def _show_url(self):
         from dvc.api import get_url
         from dvc.ui import ui
 
-        try:
-            url = get_url(self.args.path, repo=self.args.url, rev=self.args.rev)
-            ui.write(url, force=True)
-        except DvcException:
-            logger.exception("failed to show URL")
-            return 1
```

Comment:
This is a type of captain-obvious type of wrappers that we've been removing for awhile. Here it makes much more sense to just raise FileNotFoundError so I've removed this wrapper.